selfdrive/car/volkswagen/values.py

```diff
@@ -294,6 +294,12 @@ class CAR(Platforms):
     chassis_codes={"A3"},
     wmis={WMI.VOLKSWAGEN_USA_CAR},
   )
+  VOLKSWAGEN_JETTA_SPORTWAGEN = VolkswagenPQPlatformConfig(
+    [VWCarDocs("Volkswagen Jetta Sportwagen TDI 2011-2014")],
```

Comment:
We haven't been adding permutations for engine types, except for hybrid or electric. Let's also add the sedan.

```suggestion
    [VWCarDocs("Volkswagen Jetta 2011-2014")],
    [VWCarDocs("Volkswagen Jetta Sportwagen 2011-2014")],
```